e2e/07_router_standalone.spec.ts

```diff
@@ -97,4 +102,6 @@ test.describe('07_router standalone', () => {
     await testRouterExample(page, port);
     cp.kill();
   });
+
+  // FIXME shouldn't we also test `--with-ssr`?
```

Comment:
@himself65 How does that sound?
Related: https://github.com/dai-shi/waku/pull/420